oqupy/bath_dynamics.py

```diff
@@ -91,47 +119,70 @@ def bath_occupation(self,
         ----------
         freq : float
             Frequency about which to calculate the change in occupation.
-        dw : tuple (default = (1.0,1.0)):
+        dw : float (default = 1.0)
             Bandwidth of about the frequency to calculate the energy within. By
             default what is returned by this method is a *density*.
+        change_only : bool (default = False)
+            Option to include the initial occupation in the result.
         Returns
         -------
         times : List[float]
 
         bath_energy: float
         """
-        corr_mat_dim = len(self._process_tensor.times)
+        corr_mat_dim = len(self._process_tensor)
         current_corr_dim = self._system_correlations.shape[0]
-        correlation_set = [(x,y) for y in range(current_corr_dim,corr_mat_dim)\
-                           for x in range(y+1)]
-        if len(correlation_set)>0:
-            dim_diff = corr_mat_dim-self._system_correlations.shape[0]
-            coup_op = self.bath.coupling_operator
-            _new_sys_correlations = \
-                self._process_tensor.calc_correlations(coup_op,correlation_set)
+        dt = self._process_tensor.dt
+        last_time = corr_mat_dim*dt
+        tlist = np.arange(0,last_time+dt,dt)
```

Comment:
Minor thing on style: I think one is supposed to put whitespaces after commas.
`tlist = np.arange(0,last_time+dt,dt)` -> `tlist = np.arange(0, last_time+dt, dt)`

This concerns a lot of lines in the code, would be great if you have a rough look through. Note: Editors sometimes have an auto-formatting function that might help. The vscode editor has.
I shall configure pylint accordingly (I haven't found out why it doesn't complain about this at the moment).

Reply:
The auto-formatting in Spyder made some changes I didn't like so I've gone through by hand and made quite a few changes, hopefully caught them all!

Reply:
That's great. Thank you!